### ISSUE TITLE
Ability to Set the table name in AddTable

### ISSUE THREAD

**Issue by jjmartin** [360EntSecGroup-Skylar/excelize#216]
I have an excel template that I am manipulating and I have pivot tables referring to a named table, 
If I add table to the sheet where an existing table is, then the data may expand beyond the length of the existing table and the pivot tables miss it.

the next bet i have is to add a table to the data I am adding, but I don't have the ability to set the name of the table and so the pivot tables lose the reference.

Basically i just want to be able to send in a parameter to this line here instead of it being auto-created:
https://github.com/360EntSecGroup-Skylar/excelize/blob/master/table.go#L125

**Comment by jjmartin** [comment]
thanks so much for jumping on this - it does exactly what i asked for but unfortunately doesn't solve my problem, creating the new table doesn't seem to keep the link to the pivot tables in the document.

Would be nice to be able to alter the range of an existing table maybe.   Some of this excel internal linking seems like magic.
